### ISSUE TITLE
no-unused-collections should not fire if collection is exported

### ISSUE THREAD

**Issue by flunderpero** [SonarSource/eslint-plugin-sonarjs#197]
**I want to report a bug.**

Exported members of a module / class should be ignored.

**Reproducer**

```javascript

export collection = new Map()
```
